components/automate-ui/src/app/pages/infra-proxy/chef-servers-list/chef-servers-list.component.html

```diff
@@ -0,0 +1,58 @@
+<div class="content-container">
+  <div class="container">
+      <main>
+        <chef-page-header>
+          <chef-heading>Chef Servers</chef-heading>
+          <chef-subheading>Manage Chef Servers with Chef Automate.</chef-subheading>
+        </chef-page-header>
+
+        <app-create-chef-server-modal
+          [visible]="createModalVisible"
+          [creating]="creatingChefServer"
+          [createForm]="createChefServerForm"
+          (close)="closeCreateModal()"
+          [conflictErrorEvent]="conflictErrorEvent"
+          (createClicked)="createChefServer()">
+        </app-create-chef-server-modal>
+        <app-delete-object-modal
+          [visible]="deleteModalVisible"
+          objectNoun="server"
+          [objectName]="serverToDelete?.name"
+          (close)="closeDeleteModal()"
+          (deleteClicked)="deleteServer()"
+          objectAction="Delete">
+        </app-delete-object-modal>
+        <div class="page-body">
+          <chef-toolbar>
+            <chef-button primary (click)="openCreateModal()">Add Chef Server</chef-button>
+          </chef-toolbar>
+          <chef-table-new>
+            <chef-table-header>
+              <chef-table-row>
+                <chef-table-header-cell>Chef Server Name</chef-table-header-cell>
```

Comment:
- Replace `Chef Server Name` with `Name`.
- The Description should be a placeholder of Name.
- The Name should be treated as an ID(unique).

Ref: Team/Project Create Model popup.
![project-create-model](https://user-images.githubusercontent.com/44496937/70385770-4d4f3400-19b6-11ea-9624-7c91d273a663.png)